src/config/lib_package.rs

```diff
@@ -66,8 +66,8 @@ impl LibPackage {
 
         let wasm_file = {
             let source = metadata
+                //Can't use absolute path because the path gets stored in snapshot testing, and it differs between developers
```

Comment:
Rust fmt: https://doc.rust-lang.org/book/ch03-04-comments.html#:~:text=In%20Rust%2C%20the%20idiomatic%20comment,%23!%5B